code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -417,7 +415,326 @@
 				/obj/structure/constructshell = 50 //Fuck it we ball what could go wrong
 				)
 
+/obj/effect/spawner/lootdrop/trade_sol/federation_minor
+	name = "11. trans-solar federation small gear"
+	loot = list(
+				/obj/item/storage/box/enforcer_rubber = 50,
+				/obj/item/storage/box/buck = 30, //Only seven shots, make them count
```

Comment:
im not saying to remove this, but i am here to say its a terrible idea

Reply:
Buckshot is indeed a terrible idea. I'm going to swap it to make it one of the more big-ticket items and move the dispenser to common